### PR TITLE
Metric: added meanAbsoluteError

### PR DESCRIPTION
**Pull request by dcrescim**
I wrote a simple **meanAbsoluteValue** metric that we'll need. I'm gonna start throwing more metrics in this PR until we have most of the ones on this list.

https://scikit-learn.org/stable/modules/classes.html#module-sklearn.metrics

I'll start with the basic ones (meanSquaredError, accuracy, and then we'll work our way down to some of the strange ones).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "danfojs-node": "0.3.0"
   },
   "scripts": {
-    "test": "nyc mocha --require ts-node/register test/**/**/*.test.ts",
+    "test": "nyc mocha --require ts-node/register 'test/**/**/*.test.ts'",
```
Comment (risenW) [inline package.json:18]: I see quotes around the test script. Any particular reason why?/

Comment (dcrescim) [reply package.json:18]: This is literally just so mocha finds all the test files. If you don't have the single quotes, then mocha only  finds test files that are **exactly** `3` levels deep instead of `n` levels deep. 
Stackoverflow said the answer was to throw single quotes around it and it worked.

The easiest way to see that for yourself is to remove them, and you'll see that the number of tests goes down to 26 (cause it misses all the utils tests), but if you have them then the total test suite is 40.

Comment (risenW) [reply package.json:18]: Oh, interesting. Thanks! learnt that one.
